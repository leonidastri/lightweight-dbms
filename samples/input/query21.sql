select B.D, B.E, S2.A, S1.B from Sailors S1, Boats B, Sailors S2 where S1.A = B.E AND S2.A = S1.A;