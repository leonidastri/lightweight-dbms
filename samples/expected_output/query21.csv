103,1,1,200
101,2,2,200
107,2,2,200
102,3,3,100